enginetest/memory_engine_test.go

```diff
@@ -183,26 +183,31 @@ func TestSingleQueryPrepared(t *testing.T) {
 
 // Convenience test for debugging a single query. Unskip and set to the desired query.
 func TestSingleScript(t *testing.T) {
-	t.Skip()
```

Comment:
Go ahead and tidy this up so it's nice and clean and ready for the next person who needs to use it (including removing all the commented out code below)